web-server/v0.4/src/utils/parse.js

```diff
@@ -300,3 +316,74 @@ export const generateIterationClusters = (config, iterations) => {
 
   return parseClusteredIterations(primaryMetricIterations, clusterLabels, selectedConfig);
 };
+
+export const getComparisonColumn = (record, iterationNo) => {
+  const children = [];
+  for (let i = 0; i < iterationNo; i += 1) {
+    children.push({
+      title: `Iteration - ${i}`,
+      dataIndex: '0',
```

Comment:
In general, we should set unique values here for `dataIndex` and `key` since we are iterating over the length of iterations for a cluster.